docs/source/working.rst

```diff
@@ -0,0 +1,530 @@
+The script relies on a template folder which a reference "fst" file. The folder is duplicated, files are created for each wind turbine operating conditions wind speed/rpm), OpenFAST is run, and the linearization files are postprocessed.
+
+
+The script `runCampbell` generates either a set of CSV files or an Excel file. The script attempts to identify the modes (for instance 1st tower fore-aft mode, 1st flap mode, etc.), but a manual process is usually required to fully identify the mode. This process can be difficult and tedious. It is recommended to proceed first with simulations in vacuum, and with few operating points, to get familiar with the system.
+
+The manual identification process consists in changing the CSV file `Campbell_ModesID.csv` (or the Excel spread sheet `ModesID` if Excel output is used). To avoid having this file rewritten when rerunning `runCampbell`, it is recommended to rename this file as `Campbell_ModesID_Manual.csv`.  The part of the script `runCampbell` that plots the Campbell diagram can be adjusted so as to use the "Manual" file. 
+It is recommended to use the CSV format since this is the method compatible with Python and MacOS.
+
+The manual identification process consists in attributing indexes in the table of modes, where the index corresponds to the list of sorted mode frequencies.
+
+For instance, opening the CSV file in excel, the `ModeID` file might look as follows:
+
+.. code::
+
+    Mode Number Table      
+    Wind Speed (mps)   2.0   5.0   8.0
+    1st Tower FA        0     0     0
+    1st Tower SS        1     0     0
+
+In this example, we  assume that linearizations were run at 2, 5 and 8m/s. "0" in the table indicates that a mode was not identified. You can look at the file `Campbell_Summary.txt` to have a look at the frequencies, damping and "modal content" for each mode and operating point. For more details, you can open the individual CSV files for each operating point. (If you used the Excel format, these are in different sheets).
+You might find that for 2 and 5m/s, the tower Fore-Aft is the second frequency, and the tower side-side is the first frequency that shows up in the list of modes. At 8m/s you might find that the opposite occurs. In that case, you will edit the file such that it is as follows:
+
+.. code::
+
+    Mode Number Table      
+    Wind Speed (mps)   2.0   5.0   8.0
+    1st Tower FA        2     2     1
+    1st Tower SS        1     1     2
+
+
+The main question is how to determine which mode is which. There is no true solution to this question, here are some elements to help the identifications: 
+
+ - The system frequencies are usually easy to determine at 0 m/s and 0 rpm. The system frequencies will vary progressively from this reference point as the RPM/WS/pitch changes. Blade modes will typically display a "splitting" equal to +/- the rotational speed frequency as the rotational speed increases.
```

Comment:
The progressive and regressive (forward and backward whirling) modes have this +/- speed splitting.  The collective mode in flap tends to increase with rotor speed due to centrifugal stiffening.